### ISSUE TITLE
Tmuxinator not opening 3rd pane

### ISSUE THREAD

**Issue by rightaway** [tmuxinator/tmuxinator#351]
```
  - project:
      root: ~/project
      layout: c4a8,272x63,0,0{111x63,0,0[111x48,0,0,27,111x14,0,49,32],160x63,112,0,33}
      panes:
        - ls -la
        - npm run start
        - vim
```

Sometimes with the window configuration above, tmuxinator doesn't open the 3rd pane (vim). Maybe it's because the 2nd pane (npm run start) gives an error because the database server hasn't started, because I start tmuxinator immediately upon logging in.

```
Knex:Error Pool2 - Error: connect ECONNREFUSED 127.0.0.1:5432
Knex:Error Pool2 - Error: connect ECONNREFUSED 127.0.0.1:5432
```

But that shouldn't prevent tmuxinator from opening all 3 panes.

**Comment by DanielArndt** [comment]
I'm also having a similar issue with a custom layout. The third pane never opens, and I get a `create pane failed: pane too small` error.

**Comment by podung** [comment]
I believe this should be fixed by #375.  @rightaway if upgrading to 0.7.2 doesn't fix your issue please re-open.
